test/perf/perf_commitlog.cc

```diff
@@ -42,6 +43,27 @@ struct test_config {
     uint64_t max_flush_delay_in_ms;
 };
 
+struct clperf_result : public perf_result {
+    double aio_writes;
+    double aio_write_bytes;
+
+    clperf_result()
+        : aio_writes(engine().get_io_stats().aio_writes)
+        , aio_write_bytes(engine().get_io_stats().aio_write_bytes)
+    {}
```

Comment:
We can add these to the standard thing, it's generic enough.

Reply:
You could, but then the next user comes and thinks "I wanted aio_reads as well..."
Better approach might be to add them as a shared sub-type in perf.hh (maybe via mix-in)

Reply:
Made it a mixin + extended shared type. Can be used by any bench if needed. CL now just type-defs + fixes its json output.